source/digits_hits/include/GateJPETActor.hh

```diff
@@ -0,0 +1,197 @@
+#include "GateConfiguration.h"
+
+#ifndef GATEJPETACTOR_HH
+#define GATEJPETACTOR_HH
+
+#include "TROOT.h"
+#include "TFile.h"
+#include "TNtuple.h"
+#include "TTree.h"
+#include "TBranch.h"
+#include "TString.h"
+
+#include "GateVActor.hh"
+#include "GateJPETActorMessenger.hh"
+
+struct iaea_header_type;
+struct iaea_record_type;
+
+/**
+ * Actor created for JEPT use
+ * @author: Mateusz Bała
+ * @email: bala.mateusz@gmail.com
+ */
+class GateJPETActor : public GateVActor
+{
+public:
+	virtual ~GateJPETActor();
+	// This macro initialize the CreatePrototype and CreateInstance
+	FCT_FOR_AUTO_CREATOR_ACTOR(GateJPETActor)
+	// Constructs the sensor
+	virtual void Construct();
+	// Callbacks
+	virtual void UserSteppingAction(const GateVVolume *, const G4Step*);
+	virtual void PreUserTrackingAction(const GateVVolume *, const G4Track*);
+	virtual void BeginOfEventAction(const G4Event * e);
+
+	/// Saves the data collected to the file
+	virtual void SaveData();
+	virtual void ResetData();
```

Comment:
Please, add a comment what ResetData() is for. SaveData() I can guess, but ResetData ?